### ISSUE TITLE
LOCALNET-EX/AG_LOCALNET-EX.txt Breakage

### ISSUE THREAD

**Issue by Cybo1927** [InnoScorpio/Public_Adblock_Filters#1]
`https://raw.githubusercontent.com/InnoScorpio/Public_Adblock_Filters/master/LOCALNET-EX/AG_LOCALNET-EX.txt` breaks internet browsing on my Pixel 3.

When I use the list it block all DNS queries, there also seems to be a problem where whitelisting the IP address doesn't fix it so all I can do is turn off the list for now. The problematic rule is `198.18.*^$network`

https://imgur.com/a/yOkBbz0

**Comment by InnoScorpio** [comment]
Try downloading the new list, to see if it resolves the problem on your Pixel 3. I will continue working on these lists making sure they are usable for most people while reducing the number of problems that can occur. I made a new list where DNS queries are not being blocked. I tested it out
I made the following changes to the new list as 04.25.2020:
-  `198.18.*^$network`, `198.19.*^$network`,  `[*]^$network`, `[100:]^$network`, `[::ffff:0:*]^$network` rules that are removed from `AG_LOCALNET-EX.txt`. 
- I disable localhost and deprecated IP addresses that can be useful but depending on the user choices to enable them if they don't use deprecated and localhost IP addresses if they want Adguard to filter localhost traffic. These are optional.
-  Common exceptions to the list are added to makes the list usable for most people. 
- It is recommended to make your own list if you want to block Mobile ISP and Home ISP wide-area-network addresses. Mobile ISP and Home ISP WAN addresses are rarely used so they should be blocked as well.
